src/composables/trade/useGnosis.ts

```diff
@@ -350,9 +351,15 @@ export default function useGnosis({
             if (
               priceQuote.status === 'fulfilled' &&
               priceQuote.value &&
-              priceQuote.value.amount != null
+              (exactIn.value
```

Comment:
This could be simplified to `const amount = exactIn.value ? priceQuote.value.buyAmount : priceQuote.value.sellAmount` and use that in both places.

Reply:
Thanks, refactored this entire block of code.